charts/bitwarden-eso-provider/templates/tests/test-connection.yaml

```diff
@@ -23,4 +24,4 @@ spec:
       configMap:
         name: test-connection
         defaultMode: 0777
-  restartPolicy: Never
+  restartPolicy: Always
```

Comment:
Not really, but im not sure how to make the configmap continuously poll until `success`, otherwise it just fires once before the main pod is ready and never again 🤷‍♀️

Reply:
wait i think `OnFailure` will do the trick 🥇